src/Documentation/Markdown/Markdown.js

```diff
@@ -127,47 +127,58 @@ export default class Markdown extends Component {
   }
 
   render() {
-    const { markdown, githubLink, section, file, onFileSelect } = this.props
+    const { markdown, githubLink, section, file, onFileSelect, load } = this.props
     const files = sidebar[section].files
     const fileIndex = files.findIndex(f => f === file)
     const showPrev = fileIndex > 0
     const showNext = fileIndex + 1 < sidebar[section].files.length
 
-    return (
-      <Content>
-        <GithubLink href={githubLink} target="_blank">
-          <i /> Edit on Github
-        </GithubLink>
-        <ReactMarkdown
-          key={`${section}-${fileIndex}`}
-          className="markdown-body"
-          escapeHtml={false}
-          source={markdown}
-          renderers={{
-            code: CodeBlock,
-            heading: HeadingRenderer,
-            virtualHtml: HtmlRenderer
-          }}
-          astPlugins={[linker()]}
-        />
-        <NavigationButtons>
-          <Button
-            onClick={() => onFileSelect(files[fileIndex - 1], section)}
-            disabled={!showPrev}
-          >
-            <i className="prev" />
-            <span>Prev</span>
-          </Button>
-          <Button
-            onClick={() => onFileSelect(files[fileIndex + 1], section)}
-            disabled={!showNext}
-          >
-            <span>Next</span>
-            <i className="next" />
-          </Button>
-        </NavigationButtons>
-      </Content>
-    )
+    if(load){
+      return(
+        <Content>
+          <div style={{width:'100%',height:'100%',display:'flex',alignItems:'center',justifyContent:'center',flexDirection:'column'}}>
```

Comment:
make a separate component, move CSS out of it - the same way we elsewhere

Reply:
show this component using similar approach to `isSectionActive` below instead of a global if and repeating Content twice